launch/client.launch

```diff
@@ -3,14 +3,15 @@
   <arg name="host" />
   <arg name="ns" default="quanergy" />
   <arg name="topic" default="points" />
+  <arg name="frame" default="quanergy" />
```

Comment:
```suggestion
  <arg name="frame" default="$(arg ns)" />
```
We could default to the old behavior while still giving the option to change...

Reply:
Absolutely reasonable changed I updated it in my PR. @rossctaylor